pkg/server/server.go

```diff
@@ -0,0 +1,150 @@
+package server
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"time"
+
+	database "github.com/sachintiptur/http-app/pkg/util"
+)
+
+type DbStruct struct {
```

Comment:
That name is not really helpful. The DBStruct is what eventually implements ServeHTTP so maybe it's more a Server?